src/hotspot/cpu/arm/sharedRuntime_arm.cpp

```diff
@@ -1505,17 +1505,15 @@ void SharedRuntime::generate_deopt_blob() {
   // interpreter would need. So this stack banging should never
   // trigger a fault. Verify that it does not on non product builds.
   // See if it is enough stack to push deoptimized frames
```

Comment:
Now the two comment blocks flow together, please add a period to line 1507. Thanks.

Reply:
Done. Thanks for pointing this out.